### ISSUE TITLE
A way to choose implementation

### ISSUE THREAD

**Issue by Tylwen** [OsipXD/MimicAPI#27]
Allow you to choose the experience you want.
Between MMOCore and Vanilla for example
Currently, MMOCore takes over without being able to be modified

**Comment by osipxd** [comment]
Good idea. I thought about it.
Config like this:
```yaml
experience: vanilla
classes: mmocore
disabled-items-registries:
  - mmoitems
  - itemsadder
```

**Comment by osipxd** [comment]
Depends on #31

**Comment by osipxd** [comment]
Config format:

```yml
# Mimic v0.7
#
# Here you can configure Mimic APIs.
# Use command '/mimic info' to list available implementations.

# [LevelSystem API]
# Specify here the preferred level system implementation.
# If the value is empty, will be used implementation with the highest priority.
level-system: ''
# [ItemsRegistry API]
# List here disabled items registry implementations.
# You can't disable 'mimic' and 'minecraft' items registries.
# By default, all implementations are enabled.
disabled-items-registries: []
# [ClassSystem API]
# Specify here the preferred class system implementation.
# If the value is empty, will be used implementation with the highest priority.
class-system: ''

```

**Comment by osipxd** [comment]
@Tylwen, this feature was implemented in Mimic v0.7. Please check it works as you want
